controllers/ssp_controller.go

```diff
@@ -205,7 +205,14 @@ func (r *sspReconciler) Reconcile(ctx context.Context, req ctrl.Request) (res ct
 	}
 
 	if r.areCrdsMissing {
-		err := updateStatusMissingCrds(sspRequest, r.crdList.MissingCrds())
+		var missingCrds []string
+		if !sspRequest.Instance.Spec.FeatureGates.DeployTektonTaskResources {
+			missingCrds = excludeTektonCrd(r.crdList.MissingCrds())
+		} else {
+			missingCrds = r.crdList.MissingCrds()
+		}
```

Comment:
```
missingCrds := r.crdList.MissingCrds()
if !sspRequest.Instance.Spec.FeatureGates.DeployTektonTaskResources {
	missingCrds = excludeTektonCrd(missingCrds)
}
```

?

Reply:
I removed this code, as I didn't pay attention to the `setup.go` where there is also error log of missing crds.